org/glassfish/mojarra-parent/maven-metadata.xml

```diff
@@ -9,6 +9,6 @@
       <version>2.3.9.payara-p2</version>
       <version>2.3.9.payara-p3</version>
     </versions>
-    <lastUpdated>20190812101840</lastUpdated>
+    <lastUpdated>20200214104640</lastUpdated>
```

Comment:
So today at 10:46:40 you build your own version. Then it is logical it has different hashes, because content is different (ie manifest headers). :)